.mvn/extensions.xml

```diff
@@ -0,0 +1,7 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
+  <extension>
+    <groupId>io.jenkins.tools.incrementals</groupId>
+    <artifactId>git-changelist-maven-extension</artifactId>
+    <version>1.0-beta-7</version>
```

Comment:
Did you try run the incrementalify maven plugin command from the incremental repo readme? You’re using a very old version here

Reply:
yep, I try to deploy to the repo (`mvn -Dset.changelist -DskipTests clean deploy`), I do not have permissions so I have tested it with a local nexus repo.

Reply:
changed to version 1.2